Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
+        return number_1 % number_2
     else:
         raise Exception('This operator is invalid. Try running the program again.')
```

Comment:
Adding a error checker for wrong number inputs will help make sure the code doesn't break. Currently, I see that the code only handles when the wrong operation is inputted

Reply:
I agree. Perhaps adding  a try except block to catch any errors to prevent program from crashing.